test/spec/todo-item/todo-item.spec.ts

```diff
@@ -90,4 +90,13 @@ describe('POST /todos', () => {
         .to.equal('The title is empty or the title is not a string.')
     }
   })
+
+  describe('GET /todos', () => {
```

Comment:
We should have following tests under this suite:
- should return empty array when there are no todo items
- should return and match the todo id from the database when there are todo items present
- should return todos in the order they were created